util/udev/50-qmk.rules

```diff
@@ -60,3 +60,6 @@ SUBSYSTEMS=="usb", ATTRS{idVendor}=="239a", ATTRS{idProduct}=="000e", TAG+="uacc
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="2a03", ATTRS{idProduct}=="0036", TAG+="uaccess", ENV{ID_MM_DEVICE_IGNORE}="1"
 ### Micro
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="2a03", ATTRS{idProduct}=="0037", TAG+="uaccess", ENV{ID_MM_DEVICE_IGNORE}="1"
+
+# hid_listen
+KERNEL=="hidraw*", MODE="0660", GROUP="plugdev", TAG+="uaccess", TAG+="udev-acl"
```

Comment:
Is there a concrete reason for `GROUP=plugdev`?
It is not set for the other rules and is less secure than `TAG=uaccess`

Reply:
I don't remember where I found that, but it was in the original rule I copied. I didn't have a specific reason to add that, if there's a better configuration that could be changed.

Reply:
`TAG+="uaccess"` should be enough. It makes sure that the currently locally present user has access to the device node.
`udev-acl` may be the old version of that.

Given that all the other rules in this file only use `TAG+="uaccess"` that probably also works here.

Reply:
When `GROUP=` is removed, `MODE=` is also not necessary anymore.

Reply:
Would you like to submit that PR? You seem to know a lot more about udev rules than I do and it's always good to get more knowledgeable hands on it.

Reply:
Sure